pkg/webui/lib/shared-messages.js

```diff
@@ -294,6 +294,9 @@ export default defineMessages({
   refresh: 'Refresh',
   replaceWebhook: 'Replace webhook',
   removeCollaborator: 'Remove collaborator',
+  requireAuthenticatedConnection: 'Require authenticated connection',
+  requireAuthenticatedConnectionDescription:
+    'This gateway may only connect if it uses an authenticated Basic Station or MQTT connection',
```

Comment:
```suggestion
    'Controls whether this gateway may only connect if it uses an authenticated Basic Station or MQTT connection',
```

This needs to be reworded to avoid confusion.

Reply:
Actually the wording of all checkbox descriptions are misleading. I'll open an issue about this.